includes/Stories_Renderer/Generic_Renderer.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ * Generic_Renderer class.
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\Stories_Renderer;
+
+use Google\Web_Stories\Embed_Base;
+
+/**
+ * Generic_Renderer class.
+ */
+class Generic_Renderer extends Renderer {
+
+	/**
+	 * Perform initial setup for object.
+	 *
+	 * @return void
+	 */
+	public function init() {
+
+		parent::init();
+
+		$this->assets();
+	}
+
+	/**
+	 * Enqueue assets.
+	 *
+	 * @return void
+	 */
+	public function assets() {
+
+		parent::assets();
+
+		if ( $this->is_view_type( 'grid' ) && ! $this->is_amp_request() && true !== $this->attributes['show_story_poster'] ) {
+			wp_enqueue_style( Embed_Base::STORY_PLAYER_HANDLE );
```

Comment:
Should we use [Assets](https://github.com/google/web-stories-wp/blob/main/includes/Traits/Assets.php) trait here?